### ISSUE TITLE
Captions not subtitles

### ISSUE THREAD

**Issue by LJWatson** [alphagov/accessibility-guidance#25]
In [Understanding WCAG](https://github.com/alphagov/accessibility-guidance/wiki/Understanding-WCAG-2.0) it uses the word "subtitles". For accessibility purposes the correct word is "captions" - and almost all of the information found online about this topic will refer to them as such.

Note: internationally subtitles are provided for people who do not speak the language of the original content, captions are provided for people who do speak that language but are unable to hear it.

**Comment by accessiblewebuk** [comment]
I think the difficulty here is that across Europe the term subtitles is synonymous with captions, whereas elsewhere subtitles does imply different language. Even the BBC predominantly use the term subtitles instead of captions. For example http://www.bbc.co.uk/rd/projects/live-subtitle-quality http://www.bbc.co.uk/aboutthebbc/insidethebbc/howwework/policiesandguidelines/subtitles.html and in particular the BBC subtitle guidlelines http://www.bbc.co.uk/aboutthebbc/insidethebbc/howwework/policiesandguidelines/subtitles.html

Obviously we need people to understand the relationships in things like WCAG which do refer to captions, but we need to be aware that most people in the UK probably think of subtitles for Deaf and hard of hearing people, and are much less famliar with the term captions.

**Comment by spencerball** [comment]
In this case, it might be useful to refer to both 'subtitles' and 'captions'. As Richard says, people are less familiar with the term 'captions', but that's the term used more widely in accessibility guidance (including the WCAG pages we link to from the relevant bullet points in our guidance).

On GOV.UK, we sometimes use single quotes for words that are usually called something else. So we could write here, for example:

You should provide:
* subtitles (or 'captions') for pre-recorded video with sound - this should include all dialogue and important sound effects ([success criterion 1.2.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-captions.html))

**Comment by LJWatson** [comment]
I would say something like "captions (sometimes known as subtitles in the UK)...". WCAG uses "captions" and that is the standard we're pointing people to, so I think this is the simplest way of capturing both ideas.
